examples/game_of_life/src/preset.rs

```diff
@@ -1,6 +1,7 @@
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[derive(Debug, Clone, Copy, Default, PartialEq, Eq)]
 pub enum Preset {
     Custom,
+    [default]
```

Comment:
Pretty sure this should be

```suggestion
    #[default]
```